### ISSUE TITLE
Could not start the 4.0.0-beta1 container

### ISSUE THREAD

**Issue by cobolbaby** [distribworks/dkron#1461]
**Describe the bug**

**To Reproduce**

```yaml
version: "2.3"
services:
  worker:
    container_name: dkron
    hostname: dkron
    image: dkron/dkron:4.0.0-beta1-light
    restart: always
    cpu_count: 1
    mem_limit: 2g
    logging:
      driver: "json-file"
      options:
        max-size: 2m
    network_mode: host
    # environment:
    #   TZ: Asia/Shanghai
    volumes:
      # - dkron-resource:/opt/dkron-resource:rw
      - /etc/localtime:/etc/localtime:ro
      # - ./volume/dkron/data:/var/lib/dkron/data
      - /var/run/docker.sock:/var/run/docker.sock
    command: |
      agent
      --server
      --log-level debug
      --tag docker-cli=true
      --tag python=true
      --bind-addr localhost:6231
      --rpc-port 6232
      --enable-prometheus
      --disable-usage-stats
      --bootstrap-expect=1
      --http-addr :6230
      --data-dir /var/lib/dkron/data

# networks:
#   default:
#     external: true
#     name: infra

# volumes:
#   dkron-resource:      
#     name: dkron-resource
```

```bash
$ docker-compose up

Pulling worker (dkron/dkron:4.0.0-beta1-light)...
4.0.0-beta1-light: Pulling from dkron/dkron
4abcf2066143: Already exists
5b0ecaac30bf: Pull complete
5de4d443a3be: Pull complete
Recreating dkron ... done
Attaching to dkron
dkron     | time="2024-02-07T10:45:38+08:00" level=info msg="No valid config found: Applying default values." error="Config File \"dkron\" Not Found in \"[/etc/dkron /root/.dkron /config]\""
dkron     | time="2024-02-07T10:45:38+08:00" level=debug msg="starting plugin" args="[/usr/local/bin/dkron shell]" node=dkron path=/usr/local/bin/dkron
dkron     | time="2024-02-07T10:45:38+08:00" level=debug msg="plugin started" node=dkron path=/usr/local/bin/dkron pid=17
dkron     | time="2024-02-07T10:45:38+08:00" level=debug msg="waiting for RPC address" node=dkron plugin=/usr/local/bin/dkron
dkron     | time="2024-02-07T10:45:38+08:00" level=fatal msg="Unrecognized remote plugin message: time=\"2024-02-07T10:45:38+08:00\" level=info msg=\"No valid config found: Applying default values.\" error=\"Config File \\\"dkron\\\" Not Found in \\\"[/etc/dkron /root/.dkron /config]\\\"\"\nThis usually means\n  the plugin was not compiled for this architecture,\n  the plugin is missing dynamic-link libraries necessary to run,\n  the plugin is not executable by this process due to file permissions, or\n  the plugin failed to negotiate the initial go-plugin protocol handshake\n\nAdditional notes about plugin:\n  Path: /usr/local/bin/dkron\n  Mode: -rwxr-xr-x\n  Owner: 0 [root] (current: 0 [root])\n  Group: 0 [root] (current: 0 [root])\n  ELF architecture: EM_X86_64 (current architecture: amd64)\n"
```

**Expected behavior**

**Screenshots**

** Specifications:**
 - OS: [e.g. linux]
 - Version [e.g. 2.0.1]

**Additional context**

**Comment by vcastellm** [comment]
Something look messed up with the binaries in the image, I'm checking

**Comment by cobolbaby** [comment]
I'm wondering, why does a configuration initialization issue have an impact on the plugin's startup, causing this kind of error?

https://github.com/distribworks/dkron/blob/c35d31e43a8984034eb5d0dd7c38ab19d18c6e1f/cmd/plugins.go#L140-L150

**Comment by vcastellm** [comment]
Fixed in #1465 

Yeah @cobolbaby, Dkron uses hashicorp/go-plugin, plugins are binaries that runs side by side the main process, as child processes, and communicates with the parent process using HTTP gRPC connection, in this case, the shell plugin is now embeded into the main Dkron binary as a subcommand, when that subcommand was being called, an extra info message was being output from the plugin, causing the go-plugin protocol to break and not understand what the binary was saying.

Hope it makes sense.
